azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -52,6 +51,27 @@
 _IONQ_SHOTS_INPUT_PARAM_NAME = "shots"
 _DEFAULT_SHOTS_COUNT = 500
 
+IONQ_BASIS_GATES = [
+    "measure",
+    "m",
+    "cx",
+    "cz",
+    "h",
+    "reset",
+    "rx",
+    "ry",
+    "rz",
+    "s",
+    "sdg",
```

Comment:
It would be good to double check what QIR gets generated for `sdg` and `tdg`, but I think we should remove them from this list.

Reply:
Once we check how they get currently generated, we should remove them from all providers as they don't represent neither a provider specific QIR supported gate nor an Azure Quantum gate.

Reply:
yep you're right, sdg and tdg are not generated from the transpilation